### PR TITLE
Fix failing unit test

### LINKED ISSUE #476: URGENT: Fix failing unit test on frontend

### PR DESCRIPTION
**Pull request by KhalidUmuzi**
Related issues: [please specify]

## Description:

What are you up to? Fill us in :)

## I solemnly swear that:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] I have commented my code in hard-to-understand areas
- [X] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by FaithMo — APPROVED** [review]
looks good

**Review by Sbonelo01 — APPROVED** [review]

File: frontend/src/components/pages/TeamDashboard/Presentation.jsx [hunk 1]
```diff
@@ -72,9 +72,11 @@ export default function ({ team, activityLogDayCounts }) {
     activityLogDayCounts,
   });
 
+  const dateFilter = new Date();
```
Comment (sheenarbw) [inline frontend/src/components/pages/TeamDashboard/Presentation.jsx]: If I ask you to explain to me in English what the date filter is, you will say it is the "end date" or the "current date". It's used for filtering, but what is it? It's definitely not a filter. A filter is a function.

File: frontend/src/components/pages/TeamDashboard/utils.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-export function updateActivityLogDayCounts({ activityLogDayCounts }) {
+export function updateActivityLogDayCounts({ activityLogDayCounts }, dateFilter) {
```
Comment (sheenarbw) [inline frontend/src/components/pages/TeamDashboard/utils.js]: Why use positional arguments? Isn't destructuring generally much safer? And it's good to be consistent. 

`export function updateActivityLogDayCounts({ activityLogDayCounts, dateFilter }) `

**Review by sheenarbw — CHANGES_REQUESTED** [review]
This works but the code has some strangeness to it.  

If we introduce a new convention to the code then it's good to know that the juniors will follow that convention. The main thing here is that we should aim to use restructuring and avoid using positional arguments. And when in doubt be consistent. 

The name is a bit of a bunny one - if I just see how the function is used then it is not at all clear how the date is used in the filter, is it a start date or an end date? If I want to understand a function call then I need to go sniff around in the function definition. Ideally if I look at a line of code I should understand it's intention without going on any treasure hunts. 

If I put this code in I would probably end up cleaning it up or getting someone else to.

**Review by Sbonelo01 — APPROVED** [review]

**Review by FaithMo — APPROVED** [review]

**Review by sheenarbw — APPROVED** [review]

**Pull request merged**